packages/api/internal/handlers/teams.go

```diff
@@ -34,10 +35,18 @@ func (a *APIStore) GetTeams(c *gin.Context) {
 
 	teams := make([]api.Team, len(teamsDB))
 	for i, teamDB := range teamsDB {
+		apiKey, err := team.CreateAPIKey(ctx, a.db, teamDB.ID, userID, "CLI login/configure")
```

Comment:
what if reimplement the logic in CLI and then just return here to update CLI? You can't run in pipeline and if you run it manually it's probably okay with reasonable error